src/pages/item/ItemInfo.jsx

```diff
@@ -1,39 +1,121 @@
 import React, { useState, useEffect } from "react";
-import { data } from "../../data/data";
+import Swiper from "react-id-swiper";
 import { BiMinus, BiPlus, BiShoppingBag } from "react-icons/bi";
 import { FiChevronLeft } from "react-icons/fi";
 import { useParams } from "react-router";
+import { data } from "../../data/data";
 import { Link } from "react-router-dom";
+import { RiHeartAddLine } from "react-icons/ri";
+import { useFavorites } from "../../stogre/addToLikes";
 import "./ItemInfo.css";
-import { AiOutlineHeart } from "react-icons/ai";
+import { useColorChanging } from "../../stogre/colorChangeing";
```

Comment:
when you are importing libraries and assets. try to add external libraries at first lines of codes, then import you local assets, it will help you understand the code faster and better